omero/training-server/playbook.yml

```diff
@@ -305,6 +311,25 @@
         state: started
         restart_policy: always
 
+    - name: Run docker for omero-ms-zarr
+      become: yes
+      docker_container:
+        image: openmicroscopy/omero-ms-zarr:{{ omero_ms_zarr_release }}
+        name: omero_ms_zarr
+        env:
+          CONFIG_omero_db_host: localhost
+          CONFIG_omero_db_user: omero
+          CONFIG_omero_db_pass: omero
+          CONFIG_omero_db_name: omero
+          CONFIG_omero_data_dir: /OMERO
+        network_mode: host
+        exposed_ports: "8080"
```

Comment:
```suggestion
```
You don't need this since it's included in `published_ports`, and in any case it's [already in the Dockerfile](https://github.com/ome/omero-ms-zarr/blob/364057c77f7c079907653fd9d03bb41504ef9a9d/Dockerfile#L19).

Reply:
These were all leftovers from trying to get iptables to work. Do you have any idea what went wrong there? (Several docker threads showed needing to explicitly list exposed and published ports for `docker_container`, but since I never got it working, I don't know)

Reply:
By default Docker creates its own Iptables rules, which interacts with the rules creates by the iptables_raw module. There are several unsatisfactory ways of getting around this, the option used in this playbook is to prevent Docker updating the iptables rules:
https://github.com/ome/prod-playbooks/blob/13a7c298439be9aff940c7c7fd4c934cbc793a4e/omero/training-server/playbook.yml#L149-L152

Best way to add a forwarding rule is probably to register the output of the `docker_container` module which should include the IP.